frontend/vue/components/UserAccount/DeleteUserDataSection.vue

```diff
@@ -93,8 +93,9 @@ export default defineComponent({
       this.isButtonDisabled = true
       this.isModalVisible = true
     },
-    modalDeleteAction() {
-      this.isModalVisible = false
+    modalDeleteAction(ev: any) {
+      ev.preventDefault()
+      window.location.href = '/delete/account'
```

Comment:
@techtolentino @korgan00 I think I will need some help here. My idea was to modify this button with a link to make use of the redirection from the end-points but I couldn't find an element that fits correctly in the modal so I just opted for this option but probably you have a better approach.